exercises/clock/test/test_clock.c

```diff
@@ -0,0 +1,489 @@
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "vendor/unity.h"
+#include "../src/clock.h"
+
+char actual_time_text[MAX_STR_LEN];
+char actual_time_text2[MAX_STR_LEN];
+
+// general routines that tests expected result
+void test_expected_value(char *expected_clock_text, int hour, int minute)
+{
+   memset(actual_time_text, '\0', sizeof(actual_time_text));
```

Comment:
Should this be here? Shouldn't the implementation be responsible for null-terminating the string?